src/python/pants/reporting/reporting.py

```diff
@@ -59,7 +59,10 @@ def register_options(cls, register):
     register('--zipkin-trace-v2', advanced=True, type=bool, default=False,
               help='If enabled, the zipkin spans are tracked for v2 engine execution progress.')
     register('--zipkin-service-name-prefix', advanced=True, default='pants',
-      help='The prefix for service name for Zipkin spans.')
+              help='The prefix for service name for Zipkin spans.')
+    register('--zipkin-max-span-batch-size', advanced=True, default=100,
```

Comment:
Should set a type here: `.., type=int, ..`, which will cause options parsing to do some validation for you.